docs/how-to-guides/admin/configurations/standalone-keycloak.rst

```diff
@@ -3,10 +3,30 @@
 Stand-alone Keycloak: Renku configuration
 =========================================
 
-If you have your own Keycloak instance that you want to use for authentication with Renku you need to:
+If you have your own Keycloak instance that you want to use for authentication
+with Renku you need to:
 
 - Create a Realm named Renku
-- Setup the preferred parameters regarding user registration, token lifetime and activation of OTP.
-- Import the four clients from their respective json files in the `keycloak-standalone-clients <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/keycloak-standalone-clients>`_ folder after filling in the correct domain. **Note.** if you also have a standalone GitLab instance you should fill in the proper URIs in the gitlab.json file.
-- For the GitLab client copy the secret from the Credentials tab and paste it at the dedicated place in renku-values.yaml.
-- Same as above for gateway client.
+- Setup the preferred parameters regarding user registration, token lifetime and
+  activation of OTP.
+- Import the four clients from their respective json files in the `helm chart
+  keycloak configuration.
+  <https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/templates/_keycloak-clients-users.tpl>`_
+  You will need to replace the templated renku domain values in the JSON. If you are deploying against an external
+  GitLab, omit the GitLab client.
+
+The secrets between the clients defined in Keycloak must match the client
+secrets in the values file. The mapping between Keycloak clients and Renku
+values is as follows:
+
+=======================   =====================================
+Keycloak client           Renku values file
+=======================   =====================================
+``renku``                 ``.global.gateway.clientSecret``
+``renku-cli``             ``.global.gateway.cliClientSecret``
+``renku-ui``              ``.global.uiserver.clientSecret``
+``renku-jupyterserver``   ``.notebooks.oidc.clientSecret``
```

Comment:
Should we add (the optional) gitlab client secret here? I.e. it is `global.gitlab.clientSecret` in the values file and the client is called `gitlab` in keycloak.

Reply:
Also when I was checking the values.yaml file for renku for the kc clients I see that there is also stuff under `global.gateway.gitlabClientId` and this is set to `renku-ui`? Also there is `global.gateway.gitlabClientSecret`. Not sure where these come in and how they relate to `.global.uiserver.clientSecret`.

Reply:
yeah the `global.gateway.gitlabClientId` is unfortunately called `renku-ui` - this is the client id for the renku app in gitlab actually. The `global.uiserver.clientSecret` is the UI keycloak client.